deploy/charts/emqx/templates/StatefulSet.yaml

```diff
@@ -1,8 +1,4 @@
-{{ $configData := printf "%s\n%s\n%s\n%s"
-  (include (print $.Template.BasePath "/configmap.env.yaml") .)
-  (include (print $.Template.BasePath "/configmap.acl.yaml") .)
-  (include (print $.Template.BasePath "/configmap.loadedPlugins.yaml") .)
-  (include (print $.Template.BasePath "/configmap.loadedModules.yaml") .) }}
+{{ $configData := printf "%s\n%s\n%s\n%s"  (include (print $.Template.BasePath "/configmap.env.yaml") .)  (include (print $.Template.BasePath "/configmap.acl.yaml") .)   (include (print $.Template.BasePath "/configmap.loadedPlugins.yaml") .)   (include (print $.Template.BasePath "/configmap.loadedModules.yaml") .) }}
```

Comment:
what are the alternatives to make it multi-line so it is more readable ?

maybe assign 5 vars so the printf args list can be shorter ? 

or, maybe use multi-line values `|-`
this `$configData` is only used to generate a checksum, so the format isn't important.

Reply:
We cannot split the ``printf`` Go template function into multiple line.

[https://stackoverflow.com/questions/49816911/how-to-split-a-long-go-template-function-across-multiple-lines](url)

Reply:
let's do that.

Reply:
I refactored the `$configData` using multiple variable.